golemcli/src/commands/acl.rs

```diff
@@ -437,3 +449,29 @@ fn list(
         ))
     }
 }
+
+fn warn_if_exist(default_rule: AclRule, direction: AclRule, exist: Option<Vec<String>>) -> Option<bool> {
```

Comment:
```suggestion
fn warn_if_exist(default_rule: AclRule, direction: AclRule, exist: Option<Vec<String>>) {
```
and remove `return None;`